### DISCUSSION TITLE
Packing Lists

### DISCUSSION THREAD

**Discussion by HackHerz** [sysadminsmedia/homebox#159] [Ideas]
Hello,

I'd love to create packing lists with homebox, so i don't forget anything. Homebox could also tell me where an item is stored that i need to pack.

At the moment i do not see a feasible workaround to achive the wanted result? Additional bonus: track weight of the items and make lists public.

Anyone else that would love this feature?

**Comment by danielrosehill** [comment]
@HackHerz 

I'm on a bit of a ChatGPT kick at the moment so take what I say with a grain of salt, but as an adult ADHDer packing lists are kind of my jam (ie, I can't live without them).

I've actually developed a couple of custom GPTs for this purpose which are trained up on my "must have / do not forget" items. This could actually work pretty nicely with Homebox.

The implementation I'm thinking of would be:

-> Assign a label to your packing list items in Homebox (e.g. "Packing List Items")
-> Export that as a CSV
-> Create a custom GPT in ChatGPT along the lines of "My Custom Packing List Assistant."
-> Under knowledge, upload the packing list inventory from Homebox. Ideally you'd whittle this down to just the item name and the inventory item link

This would give you a pretty nice setup in which ChatGPT could run you through your packing list and if you don't have an item or can't remember what it looks like it could share the Homebox link to jump you to the inventory page.

Just an idea :-)

**Reply by HackHerz** [reply]
I have thougt about implementing such a thing manually using the API. The label approach seams feasible (would be nicer if labels could have values) but i want to avoid the CSV export (e.g. i notice that i forgot an item, item changes, etc would result in an outdated CSV).

My Go-Capabilites are non existent, so I'll probably end up building a static JS page that will handle all of this.

**Comment by Sjoerd82** [comment]
I currently do this in Excel, but now that I have everything in Homebox I would love to use HomeBox as a base now. Here are some great use cases:

1. LISTS. A sidebar entry. With LIST you can create all kinds of lists, among which a template packing list, a specific packing list for your next trip, and ideally both. A template as a starter. For now we can use labels, but it's slow and cumbersome, and polutes by adding extra labels.

2. Print/export the list to make a list that you can export and cross off the items you have packed.

If you have weights you can even use it to make sure you're not overburdening your backpack or know if you're staying below air travel luggage weight limits.
